s4e-backend/src/test/java/pl/cyfronet/s4e/api/OSearchControllerTest.java

```diff
@@ -83,10 +83,14 @@ public void tearDown() {
 
     private Scene buildScene(Product product, long number) throws Exception {
         JsonNode jsonNode = objectMapper.readTree(SceneTestHelper.getMetaDataWithNumber(number));
-        return SceneTestHelper.sceneWithMetadataBuilder(product, jsonNode)
+        Scene scene = SceneTestHelper.sceneWithMetadataBuilder(product, jsonNode)
                 .build();
+        scene.setSceneContent(objectMapper.readTree(SceneTestHelper.getSceneContent()));
```

Comment:
Are there any tests that check what will happened on `SceneTestHelper.getSceneContent()` equal `null`?

Reply:
test to testhelper? nope. it always return sceneContent :)

Reply:
In other words. Do you test possibility that mock of function `.getSceneContent()` on mocked (or not) `scene` object return `null`?

Reply:
in ResponseExtender there is code that checks if sceneContent is null, its now throwing exception, only logging this fact because its not fatal for application.
i added test with scene that has no sceneContent. i dont think its necessary but its a low cost.